lesson-one/examples/styles.css

```diff
@@ -1,4 +1,32 @@
 body {
-    color: white; 
+    color: #FC8EAC;
+   background-color: #D6587B; 
+   font-family: verdana;
+   font-size: 150%;
+   text-align:center;
+   
+}
+.class1{
+    border: 3px dotted #CAFFB6;
+    display: inline-block;
+}
+.class2{
+    font-family: Comic Sans MS;
```

Comment:
Very good. It's best practice to add some back up fonts, just in case the user has (rightfully) deleted Comic Sans. An example would be `"Comic Sans MS", sans-serif`